Task_TheGridProblem/index.css

```diff
@@ -1 +1,38 @@
-/* Start coding below for GRID PROBLEM TASK */
+/* Start coding below for GRID PROBLEM TASK */
+*{
+    margin: 0;
+    padding: 0;
+}
+body{
+    background-color: aquamarine;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-top: 60px;

```

Comment:
There are multiple empty lines between line 11 and line 14. These empty lines should be removed for better readability.